bowlingGame.js

```diff
@@ -0,0 +1,68 @@
+class BowlingGame {
```

Comment:
It would be nice to have something separating public and private methods in this class. A common convention in JS is to prefix all private methods with `_`, like `_isAStrike(i)`.

Especially as you are not testing those "private" methods, which is the right move, showing that they are private is useful.

There is also a way to make them really private in ES6. It's not very used but you could use it if you wanted. check https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields